sssom/util.py

```diff
@@ -402,7 +402,7 @@ def assign_default_confidence(
     if df is not None:
         new_df = df.copy()
         if CONFIDENCE not in new_df.columns:
-            new_df[CONFIDENCE] = np.NaN
+            new_df[CONFIDENCE] = 0.0  # np.NaN
```

Comment:
I dont have a problem with this for the reconciliation, but I really wouldn't want the 0% confidence to be exported in the exported mapping set - this is altering the data.

Also, this is not solving the issue where a mapping set has a `confidence` column with some missing values..

Reply:
What happens now if a mapping with a justification with  `NA` confidence (empty value) and one justification with a `0` mapping co-incide? It seems odd tha NA mappings are not set to 0 as well.

Reply:
Ok if this is the case it's fine. I would just like to generally err on the conservative side, removing only the obvious cases. Ok let's go with your solution, and also make reconcile false by default.

Reply:
`reconcile` is already `False` by default.

Reply:
This is weird, because I only noticed this problem because the reconciliation method kicked in without me adding a parameter. Can you double check that there is no place in the code that accidentally sets it to `true`?